pygpsclient/scatter_frame.py

```diff
@@ -0,0 +1,217 @@
+"""
+Scatterplot frame class for PyGPS Application.
+
+This generates a scatterplot of positions, centered on the average
+position.
+
+Created 23 March 2023
+
+:author: Nathan Michaels
+:copyright: Qualinx B.V.
+:license: BSD 3-Clause
+"""
+from collections import namedtuple
+from statistics import mean
+import math
+from tkinter import Frame, font, BOTH, YES
+from geographiclib.geodesic import Geodesic
```

Comment:
not the end of the world but I'd like to explore whether this additional dependency is really necessary. I may be underestimating the complexity but at the scale of the scatter plot (i.e. more or less a flat plane) is the heading not a case of simple geometry?

Reply:
It probably is simple geometry, but my brain didn't want to do it. Maybe I will if I get some free time later.